templates/service-postgresql.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
As discussed with Larry, we should be using an external postgre chart, and removing postgre services from this helmchart.  It needs to be HA'd, operationalized, support backups, and we want to do this in one place.  Current postgre chart is at github.com/att-comdev/openstack-helm-addons